File: pensionManagementSystem/src/app/pensioner-detail.ts
import { BankDetails } from "./bank-details";


export class PensionerDetail {
  constructor(bankDetails:BankDetails){
    this.bankDetail = bankDetails;

  }



  aadhaarNumber:any;
	name:any;
	dateOfBirth:any;
	panNumber:any;
	salary:any;
	allowance:any;
	pensionType:any;
  bankDetail:BankDetails;



}

File: pension-management-portal/src/main/java/com/cognizant/pensionmanagementportal/model/PensionerDetail.java
package com.cognizant.pensionmanagementportal.model;

import javax.persistence.Embedded;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PensionerDetail {

	private String aadhaarNumber;
	private String name;
	private String dateOfBirth;
	private String panNumber;
	private double salary;
	private double allowance;
	private String pensionType;
	@Embedded
	private BankDetail bankDetail;

}
